WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -507,5 +530,19 @@ private extension DefaultProductFormTableViewModel {
         static let noPriceWarningTitle =
             NSLocalizedString("Variations without price won’t be shown in your store",
                               comment: "Title of the no price warning row on Product Variation main screen when a variation is enabled without a price")
+
+        // Downloadable files
+        static let downloadsTitle =
+            NSLocalizedString("Downloadable files",
+                              comment: "Title of the Downloadable Files row on Product main screen")
+
+        static let emptyDownloads = NSLocalizedString("No files yet",
+                                                      comment: "Placeholder for empty Downloadable Files row on Product main screen")
+        static let singularDownloadsFormat = NSLocalizedString("%ld file",
+                                                               comment: """
+Format of the number of Downloadable Product row in the singular form. It reads, `1 file`
+""")
```

Comment:
Hi @partho-maple! Sorry, just dropping in because I noticed this. Please don't use triple quotes for `NSLocalizedString()` arguments. This currently does not work with our build system. It won't fail during local development but it will fail when we build the app for App Store deployment. 🙈 Sorry for the trouble!

Here are some guidelines about using `NSLocalizedString()`: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/docs/localization.md. That's from WordPress but we use the same build system. 🙂

Reply:
Sorry, I wasn't aware of that. Thanks for the pointer. I will remove those and make sure not to use it again.  🙂

Reply:
No problem! This is unique to us so I don't expect anyone to know about it. 🙂 And it only applies to `NSLocalizedString()` calls! Feel free to use triple quotes anywhere else. 🤪 

If you're more curious, this is the script that is used for parsing these `NSLocalizedString()` during the deployment process → https://github.com/woocommerce/woocommerce-ios/blob/develop/Scripts/localize.py